### PR TITLE
Automated PR for Unicode-TOU

### PR DESCRIPTION
**Pull request by myndzi**

### CODE REVIEW

File: src/Unicode-TOU.xml [hunk 1]
```diff
@@ -0,0 +1,174 @@
+<SPDX name="Unicode Terms of Use" identifier="Unicode-TOU" osi-approved="false">
+  <urls>
+    <url>http://www.unicode.org/copyright.html</url>
+  </urls>
+  <license>
+    <title>
+      <p>Unicode Terms of Use</p>
+    </title>
+    <optional>
```
Comment (jlovejoy) [inline src/Unicode-TOU.xml]: I am not sure this should be optional??

**Comment by jlovejoy** [comment]
(bug) lists not nesting
(legal) see comment re: first paragraph; also note that copyright in A1 is not marked up,  I think this is correct as it never was and is a specific Unicode license

File: src/Unicode-TOU.xml (hunk 1 above)
Comment (bradleeedmondson) [inline src/Unicode-TOU.xml]: agree this should not be optional. seems intended to have the legal effect of incorporating the privacy policy and tm policy. Whether it's successful, I'm not sure, but I would say don't mark as optional unless we empirically find we miss matches for licenses in the wild based on this.

**Comment by pmadick** [comment]
Agree with legal team.  Fix bug, take out optional tag and commit.

**Pull request merged**